jhipster-framework/pom.xml

```diff
@@ -48,18 +48,18 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>io.springfox</groupId>
-            <artifactId>springfox-oas</artifactId>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-webmvc-core</artifactId>
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>io.springfox</groupId>
-            <artifactId>springfox-swagger2</artifactId>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-webflux-core</artifactId>
```

Comment:
Using this dependency, I get an error when running `mvn verify` on the `webflux-mongodb` example:

```
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springdoc.core.GenericResponseService]: Factory method 'responseBuilder' threw exception; nested exception is java.lang.NoClassDefFoundError: Could not initialize class org.springdoc.core.GenericResponseService
```

If I change it to `springdoc-openapi-webflux-ui`, all tests pass. This seems to match what the [documentation recommends](https://springdoc.org/#spring-webflux-support). Therefore, I'm going to update this dependency.

Reply:
`springdoc-openapi-webflux-ui` pulls in swagger-ui webjar, we already have swagger ui at the client side.

Reply:
Yeah, `webflux-ui` causes startup issues (even though `mvn verify passes). Strangely, if I use what we have now, I get the following error on startup.

```
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 
'openApiResource' defined in class path resource 
[org/springdoc/webflux/core/SpringDocWebFluxConfiguration.class]: Unsatisfied dependency expressed through 
method 'openApiResource' parameter 2; nested exception is 
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'responseBuilder' 
defined in class path resource [org/springdoc/webflux/core/SpringDocWebFluxConfiguration.class]: Bean 
instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: 
Failed to instantiate [org.springdoc.core.GenericResponseService]: Factory method 'responseBuilder' threw 
exception; nested exception is java.lang.NoClassDefFoundError: javax/servlet/ServletException
```

Reply:
Should be fixed now.

Reply:
I spoke too soon too. 😆.
The problem is springdoc-openapi-hateoas now.
Wondering if data-rest is non-reactive only.

Reply:
fixed